src/PDS4_MESS_IngestLDD.xml

```diff
@@ -204,7 +208,7 @@
 		<DD_Value_Domain>
 			<enumeration_flag>false</enumeration_flag>
 			<value_data_type>ASCII_Short_String_Collapsed</value_data_type>
-			<pattern>[1-9]/[0-9]{9}(.[0-9]{3})?</pattern>
+			<pattern>[1-9]/[0-9]{1,9}(.[0-9]{3})?</pattern>
```

Comment:
looks like we should parse all related keywords to be the same across dictionaries. The Insight has a slightly different pattern. https://github.com/pds-data-dictionaries/ldd-insight/blob/786d16c4798e60755c50f69f94ecba72a5f9dafd/src/1.8.4.0/ldd-insight.xml#L783

as shown here: <pattern>([1-9]/)?[0-9]{1,10}(\-[0-9]{5})?</pattern>

I really don't know the best one to copy from?

Reply:
I'm not sure that it would be a good idea to use the same pattern for different projects. Projects can have significant differences in the way SCLK values counted and expressed. In this case it looks like InSight uses a dash (-) to separate the counts from fractional counts(?), whereas MESSENGER used a dot (.). Other projects had more major differences in the way their SCLKs worked. Cassini used 256ths seconds rather than milliseconds. Both Galileo and Voyager had multiple counters (a Galileo SCLKs looks like: 1/04409528:89:5:0). I think that the best approach will be to to try to insure that the patterns accurately describe valid SCLK values for a given project.